py3status/modules/net_rate.py

```diff
@@ -53,36 +63,31 @@ class Py3status:
     hide_if_zero = False
     interfaces = ''
     interfaces_blacklist = 'lo'
-    precision = 1
+    thresholds = [(0, "bad"), (1024, "degraded"), (1024*1024, "good")]
```

Comment:
The module actually switch to output the speed of the fastest interface, so there's no good way to specify the thresholds if you want something different between wifi and lan. These replicate the defaults values from before. Also, it kind of makes sense since even on wifi, if you're doing 3kbps, it's slow but if you're doing 1mpbs it's probably enough.